code/weapon/weapons.cpp

```diff
@@ -1755,6 +1754,14 @@ int parse_weapon(int subtype, bool replace, const char *filename)
 		}
 	}
 
+	if (optional_string("$Pierce Objects:")) {
+		stuff_boolean(&wip->pierce_objects);
+
+		if (optional_string("+Spawn Childs On Hit:")) {
+			stuff_boolean(&wip->spawn_childs_on_pierce);
```

Comment:
Pedantic, but should be `spawn_children_on_pierce`. And also the table entry should specify "on pierce" just to be that much clearer.